### PR TITLE
LRQA-82558

### PR DESCRIPTION
**Pull request by crislucas1**
https://liferay.atlassian.net/browse/LRQA-82558

### CODE REVIEW

**Comment by crislucas1** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.macro [hunk 1]
```diff
@@ -0,0 +1,49 @@
+definition {
+
+	macro changeScreenSize {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.macro:3]: Hey @crislucas1, I think you could modify the name of this macro. For the macro that has a `change` term in the name, it seems to me that the macro is already going to **edit** some information on the page and what you want here is that it just **"Going to simulation > clicking custom"**. So I think you could put another name like `openCustomSimulation` or `goToCustomSimulation` or another idea more following verbs that are **navigation**

**Review by ViniLopes20 — CHANGES_REQUESTED** [review]
Hey @crislucas1, you must have a commit for each macro and path created and not a single commit for the file you created. I'm going to teach you later today a really cool way to separate the code when you have a lot of modifications in the same file. I forgot to show this to you earlier. Leave to correct this PR during our sync today.

Again these are small and simple corrections. Great job again here, congratulations Cristiano 🎉

**Comment by crislucas1** [comment]
ok, thanks for the feedback @ViniLopes20

**Comment by crislucas1** [comment]
ci:test:sf

**Comment by crislucas1** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]
Very good, Christian. Congrats on working on it 🎉

**Pull request closed without merging**